chicago/bills.py

```diff
@@ -2,12 +2,14 @@
 import lxml
 import lxml.etree
 
-from pupa.scrape import Bill
+from pupa.scrape import Bill, Vote
 
 
 class ChicagoBillScraper(LegistarScraper):
     base_url = 'https://chicago.legistar.com/'
     legislation_url = 'https://chicago.legistar.com/Legislation.aspx'
+    timezone = "US/Central"
+
 
     def searchLegislation(self, search_text='', created_before=None,
```

Comment:
Yeah! Any additional params will be passed to scrape!

On Fri, Jan 16, 2015 at 10:06 AM, Forest Gregg <notifications@github.com>
wrote:

> In chicago/bills.py
> <https://github.com/opencivicdata/scrapers-us-municipal/pull/16#discussion-diff-23084807>
> :
>
> >
> >
> >  class ChicagoBillScraper(LegistarScraper):
> >      base_url = 'https://chicago.legistar.com/'
> >      legislation_url = 'https://chicago.legistar.com/Legislation.aspx'
> > +    timezone = "US/Central"
> > +
> >
> >      def searchLegislation(self, search_text='', created_before=None,
>
> Does pupa have any facilities for passing through command line args. It
> would be great to be able to specify search parameters.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencivicdata/scrapers-us-municipal/pull/16/files#r23084807>
> .
>



-- 
Paul Tagliamonte
Software Developer | Sunlight Foundation